edweiss-app/app/(app)/(tabs)/community.tsx

```diff
@@ -1,23 +1,19 @@
+import TText from '@/components/core/TText';
+import TView from '@/components/core/containers/TView';
 import RouteHeader from '@/components/core/header/RouteHeader';
 import FancyButton from '@/components/input/FancyButton';
 import { ApplicationRoute } from '@/constants/Component';
 import { router } from 'expo-router';
-import { useState } from 'react';
-import TText from '@/components/core/TText';
-import TView from '@/components/core/containers/TView';
-
-
-const ExploreTab: ApplicationRoute = () => {
-	const [pageCount, setPageCount] = useState<number>(1);
 
 
+const CommunityTab: ApplicationRoute = () => {
```

Comment:
what is the purpose of the community tab? Is it strictly a sandbox for the team?

Reply:
Hello,
For now yes, but in the future it'll be about the campus life (associations, announcements, events...).